install/helm/agones/README.md

```diff
@@ -86,8 +86,9 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.rbacEnabled`                                | Creates RBAC resources. Must be set for any cluster configured with RBAC                        | `true`                 |
 | `agones.crds.install`                               | Install the CRDs with this chart. Useful to disable if you want to subchart (since crd-install hook is broken), so you can copy the CRDs into your own chart. | `true` |
 | `agones.crds.cleanupOnDelete`                       | Run the pre-delete hook to delete all GameServers and their backing Pods when deleting the helm chart, so that all CRDs can be removed on chart deletion | `true`          |  
-| `agones.metrics.enabled`                            | Enables controller metrics on port `8080` and path `/metrics`                                   | `true`                 |
+| `agones.metrics.prometheusEnabled`                            | Enables controller metrics on port `8080` and path `/metrics`                                   | `true`                 |
```

Comment:
again I think we should have the same kind of name for prometheus and stackdriver.